### PR TITLE
Compile a version of libram for require().

### PR DESCRIPTION
**Pull request by phulin**
This will enable `require("libram")` from the Mafia command line. You could even make the following alias:

```ts
alias jsl => js { Object.assign(globalThis, require("libram")); }
```

and then:

```
> jsl AsdonMartin.installed();

Returned: false

> jsl $item`hair spray`

Returned: hair spray
name => hair spray
...
```

### CODE REVIEW

**Comment by phulin** [comment]
Unfortunately, looks like a bug in our value code block this from working somewhere. I'll have to track it down.

**Comment by phulin** [comment]
Never mind - the only issue is that we can't render the entire `libram` object because our JS -> ASH translation code can't translate enums (inconsistent typing of keys and values).

File: src/Clan.ts [hunk 1]
```diff
@@ -29,40 +29,6 @@ export interface Rank {
   id: number;
 }
 
-export class ClanError extends Error {
-  reason?: Error;
```
Comment (gausie) [inline src/Clan.ts:33]: My poor code

Comment (phulin) [reply src/Clan.ts:33]: sorry buddy, we weren't using it

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = function (api) {
+  api.cache(true);
+  return {
+    exclude: [],
+    presets: [
+      "@babel/preset-typescript",
+      [
+        "@babel/preset-env",
+        {
+          targets: { rhino: "1.7.13" },
```
Comment (gausie) [inline babel.config.js:12]: Do you want to bump this?

**Comment by phulin** [comment]
Yes

File: package.json [hunk 3]
```diff
@@ -22,13 +25,12 @@
     "dist/**/*.d.ts"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.14.8",
     "@babel/compat-data": "^7.15.0",
-    "@babel/core": "^7.15.0",
+    "@babel/core": "^7.16.12",
```
Comment (gausie) [inline package.json:29]: Presumably that bump means tha these need to be bumped to 7.17 or whatever

Comment (phulin) [reply package.json]: done

**Review by gausie — APPROVED** [review]

**Pull request merged**